src/slackmessageparser.ts

```diff
@@ -387,6 +387,15 @@ export class SlackBlocksParser {
 				retStr += "</p>";
 				return retStr;
 			}
+			case "call": {
+				const call = block as ISlackBlockCall;
+				if (call.call && call.call.v1) {
+					const { name, join_url } = call.call.v1;
+					const content = name ? name : escapeHtml(join_url);
+					return `<a href="${join_url}">${content}</a>`;
```

Comment:
please `escapeHtml()` your variables ^^ name and join_url aren't neecessarily escaped